src/main/features/claimant-response/routes/payment-date.ts

```diff
@@ -51,7 +52,7 @@ export class PaymentDatePage extends AbstractPaymentDatePage<DraftClaimantRespon
         courtOfferedPaymentIntention.paymentOption = PaymentOption.INSTALMENTS
         courtOfferedPaymentIntention.repaymentPlan = {
           firstPaymentDate: paymentPlanConvertedToDefendantFrequency.startDate,
-          instalmentAmount: paymentPlanConvertedToDefendantFrequency.instalmentAmount,
+          instalmentAmount: Math.round(paymentPlanConvertedToDefendantFrequency.instalmentAmount * 100) / 100,
```

Comment:
Do we have official guidance on what the rounding strategy should be? Javascript rounding weirdness can make this drop a penny. Some people recommend `num.toFixed(2)` (or `+num.toFixed(2)` to force it back to a number)

Reply:
i have seen Math.round used in other places so i used it not sure what rounding strategy is...i guess i am going to leave it like that for now